### PR TITLE
add map projection expression

### PR DESCRIPTION
**Pull request by jievince**
doc: https://neo4j.com/docs/cypher-manual/current/syntax/maps/#cypher-map-projection
This pr depends on
https://github.com/vesoft-inc/nebula-common/pull/596,
https://github.com/vesoft-inc/nebula-storage/pull/537

### CODE REVIEW

File: tests/admin/test_configs.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def test_configs(self):
             ['GRAPH', 'heartbeat_interval_secs', 'int', 'MUTABLE', 1],
             ['GRAPH', 'meta_client_retry_times', 'int', 'MUTABLE', 3],
             ['GRAPH', 'accept_partial_success', 'bool', 'MUTABLE', False],
-            ['GRAPH', 'system_memory_high_watermark_ratio', 'float', 'MUTABLE', 0.95],
+            ['GRAPH', 'system_memory_high_watermark_ratio', 'float', 'MUTABLE', 1.0],
```
Comment (Aiee) [inline tests/admin/test_configs.py:68]: Should revert to 0.95

File: tests/common/nebula_service.py [hunk 2]
```diff
@@ -72,7 +72,7 @@ def _format_nebula_command(self, name, meta_port, ports, debug_log=True):
         if name == 'graphd':
             params.append('--local_config=false')
             params.append('--enable_authorize=true')
-            params.append('--system_memory_high_watermark_ratio=0.95')
+            params.append('--system_memory_high_watermark_ratio=1.0')
```
Comment (Aiee) [inline tests/common/nebula_service.py:75]: ditto

File: tests/admin/test_configs.py (hunk 1 above)
Comment (jievince) [reply tests/admin/test_configs.py:68]: Why? 0.95 often fails on machine 86

File: tests/tck/features/expression/MapProjection.feature [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+      | {age: 146, key: "best"} |
+      | {age: 147, key: "best"} |
+
+  Scenario: Variable selector
+    Given a graph with space named "nba"
+    When executing query:
+      """
+      MATCH (v:player)-[:like]-(v2)
+      WHERE v.age > 45
+      RETURN v {v2}
```
Comment (CPWstatic) [inline tests/tck/features/expression/MapProjection.feature:61]: What does this means?

File: tests/tck/features/expression/MapProjection.feature [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+      | {age: 46, middleaged: true, name: "Grant Hill", playersHeLike: {age: 39, name: "Tracy McGrady"}, v2: ("Tracy McGrady" :player{age: 39, name: "Tracy McGrady"})}                                                                               |
+      | {age: 47, middleaged: true, name: "Shaquile O'Neal", playersHeLike: {age: 38, name: "Yao Ming"}, v2: ("Yao Ming" :player{age: 38, name: "Yao Ming"})}                                                                                         |
+      | {age: 47, middleaged: true, name: "Shaquile O'Neal", playersHeLike: {age: 31, name: "JaVale McGee"}, v2: ("JaVale McGee" :player{age: 31, name: "JaVale McGee"})}                                                                             |
+      | {age: 47, middleaged: true, name: "Shaquile O'Neal", playersHeLike: {age: 42, name: "Tim Duncan", speciality: "psychology"}, v2: ("Tim Duncan" :bachelor{name: "Tim Duncan", speciality: "psychology"} :player{age: 42, name: "Tim Duncan"})} |
+    # [bug]agg executed failed here now
```
Comment (CPWstatic) [inline tests/tck/features/expression/MapProjection.feature:102]: Will fix in this pr?

File: tests/tck/features/expression/MapProjection.feature (hunk 3 above)
Comment (jievince) [reply tests/tck/features/expression/MapProjection.feature:61]: It's equivalent to ` RETURN v2`

File: tests/tck/features/expression/MapProjection.feature (hunk 4 above)
Comment (jievince) [reply tests/tck/features/expression/MapProjection.feature:102]: I will make another pr to fix this problem

**Comment by jievince** [comment]
Should migrate to nebula repo

**Pull request closed without merging**